project/MongoScalaBuild.scala

```diff
@@ -32,7 +32,7 @@ object MongoScalaBuild extends Build {
   import Dependencies._
   import Resolvers._
 
-  val baseVersion = "2.2.1"
+  val baseVersion = "3.0.0"
```

Comment:
What does this number refer to?

Reply:
Good catch thats the base version - but we no longer need to do a major release.